src/app/components/normal-component/count-down/count-down.component.html

```diff
@@ -1,4 +1,14 @@
-<div>
+<div class="tile"
+     [ngStyle]="getBackgroundColor()"
+>
     <p id="remainingTime"
+       class="remainingTimeText"
+       style="margin: auto; width: 67%"
```

Comment:
à privilégier l'utilisation de classes existantes plutôt que de style=
Exemple pour avoir un boutoun "+" centré correctement avec le texte
<div class="block level-left">
<p class="level-item">Coucou voici du texte</p>
<button class="button level-item is-small" ...>+</button>
</div>

Reply:
Updated !

Reply:
hm non y'a toujours le style=margin: auto etc. !

Reply:
Fix?

Reply:
looks good à part le conflit

Reply:
résoulvdrez svp.